lib/helper.js

```diff
@@ -29,9 +29,9 @@ module.exports.helperCreatePasswordFile = function (options, params, PasswordFil
   if (!(options.password && options.passType)) {
     return false
   }
-  if (options.passType && options.passType.trim().length === 0) {
-    return false
-  }
+  //  if (!options.passType && options.passType.trim().length === 0) { // NOTE: ' ' is falsy and thus covered by !options.passType and thus by 2nd condition too

```

Comment:
Password should at last empty or 4 chars long because else will openssl error that you need a password between 4 and 1023 chars...

Reply:
Also you may add tests with empty password. To see if it is allowed ;)

Reply:
well, I think that should be covered over a new issue or through #159. make imo more sense than adding it here.

Reply:
Okay, if you here finish, let me know i will final review and merge